src/tools/csi-kata-directvolume/pkg/directvolume/nodeserver.go

```diff
@@ -223,6 +250,71 @@ func (dv *directVolume) NodeStageVolume(ctx context.Context, req *csi.NodeStageV
 	dv.mutex.Lock()
 	defer dv.mutex.Unlock()
 
+	// SPDK
```

Comment:
I found this function returns at the end of this branch. I think we should rework this function as the above, one function for SPDK, and another one for direct volume? 

```go
if req.VolumeContext[utils.KataContainersDirectVolumeType] == utils.SpdkVolumeTypeName {
    // ...
    return &csi.NodeStageVolumeResponse{}, nil
}
```

The expected one like

```
if req.VolumeContext[utils.KataContainersDirectVolumeType] == utils.SpdkVolumeTypeName {
    return SPDKXxxx(....)
} else { // if there are more conditions, feel free to use `else if (xxx)`
    // ....
}